Template/Tag/TwitterConversionTag.web.js

```diff
@@ -0,0 +1,17 @@
+(function () {
+    return function (parameters, TagManager) {
+        this.fire = function () {
+            var id = parameters.get('Pixel ID');
+            if (id) {
+                !function (e, t, n, s, u, a) {
+                    e.twq || (s = e.twq = function () {
+                        s.exe ? s.exe.apply(s, arguments) : s.queue.push(arguments);
+                    }, s.version = '1.1', s.queue = [], u = t.createElement(n), u.async = !0, u.src = '//static.ads-twitter.com/uwt.js',
+                        a = t.getElementsByTagName(n)[0], a.parentNode.insertBefore(u, a))
+                }(window, document, 'script');
+                twq('init', id);
+                twq('track', 'PageView');
```

Comment:
here in https://business.twitter.com/en/help/campaign-measurement-and-analytics/conversion-tracking-for-websites.html I don't see the conversion type "Pageview"? Only "Site visit", "Purchase", "Download", "Sign up" and "Custom". Also noticed here: https://stackoverflow.com/questions/43937596/are-there-any-different-twitter-pixel-events-than-pageview . 

I'm not sure which parameters each type supports. Only seeing `value`, `num_items` and `order_id`... Their docs are rather not so good.